tests/unit/controllers/api/v1/test_actions_rbac.py

```diff
@@ -165,6 +273,147 @@ def test_get_all_limit_larget_than_page_size(self):
+        }
+        source_ref_or_id = "%s.%s" % (ACTION_2["pack"], ACTION_2["name"])
+        clone_resp = self.__do_clone(dest_data_body,
+                                     source_ref_or_id,
+                                     expect_errors=True)
+        expected_msg = ('User "no_create_permission" doesn\'t have required '
+                        'permission "action_create" on resource "action:%s:%s"'
+                        % (ACTION_3["pack"], dest_data_body["dest_action"]))
+        self.assertEqual(clone_resp.status_code, http_client.UNAUTHORIZED)
+        self.assertEqual(clone_resp.json["faultstring"], expected_msg)
+
+    @mock.patch.object(os.path, "isdir", mock.MagicMock(return_value=True))
+    @mock.patch.object(action_validator,
+                       "validate_action",
+                       mock.MagicMock(return_value=True))
+    def test_clone_overwrite_no_destination_action_create_permission(self):
+        user_db = self.users["action_clone"]
+        self.use_user(user_db)
+        self.__do_post(ACTION_2)
+        self.__do_post(ACTION_3)
+        user_db = self.users["no_create_permission"]
+        self.use_user(user_db)
+        dest_data_body = {
+            "dest_pack": ACTION_3["pack"],
+            "dest_action": ACTION_3["name"],
+            "overwrite": True,
+        }
+        source_ref_or_id = "%s.%s" % (ACTION_2["pack"], ACTION_2["name"])
+        clone_resp = self.__do_clone(dest_data_body,
+                                     source_ref_or_id,
+                                     expect_errors=True)
+        expected_msg = ('User "no_create_permission" doesn\'t have required '
+                        'permission "action_create" on resource "action:%s:%s"'
+                        % (ACTION_3["pack"], ACTION_3["name"]))
+        self.assertEqual(clone_resp.status_code, http_client.UNAUTHORIZED)
+        self.assertEqual(clone_resp.json["faultstring"], expected_msg)
+
+    @mock.patch.object(os.path, "isdir", mock.MagicMock(return_value=True))
+    @mock.patch.object(action_validator,
+                       "validate_action",
+                       mock.MagicMock(return_value=True))
+    def test_clone_overwrite_no_destination_action_delete_permission(self):
+        user_db = self.users["action_clone"]
+        self.use_user(user_db)
+        self.__do_post(ACTION_2)
+        self.__do_post(ACTION_3)
+        user_db = self.users["no_delete_permission"]
+        self.use_user(user_db)
+        dest_data_body = {
+            "dest_pack": ACTION_3["pack"],
+            "dest_action": ACTION_3["name"],
+            "overwrite": True,
+        }
+        source_ref_or_id = "%s.%s" % (ACTION_2["pack"], ACTION_2["name"])
+        clone_resp = self.__do_clone(dest_data_body,
+                                     source_ref_or_id,
+                                     expect_errors=True)
+        expected_msg = ('User "no_delete_permission" doesn\'t have required '
```

Comment:
Please use string formatting to pass in user name instead of hard coding "no_delete_permission".